### PR TITLE
Update org references

### PR DESCRIPTION
**Pull request by jasnell**
/cc @mcollina @conoro

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -148,7 +148,7 @@ class ClinicDoctor extends events.EventEmitter {
     const stylePath = path.join(__dirname, 'visualizer', 'style.css')
     const scriptPath = path.join(__dirname, 'visualizer', 'main.js')
     const logoPath = path.join(__dirname, 'visualizer', 'app-logo.svg')
-    const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
+    // const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
```
Comment (mcollina) [inline index.js:151]: remove

File: index.js [hunk 2]
```diff
@@ -220,7 +220,7 @@ class ClinicDoctor extends events.EventEmitter {
 
     // open logo
     const logoFile = fs.createReadStream(logoPath)
-    const nearFormLogoFile = fs.createReadStream(nearFormLogoPath)
+    // const nearFormLogoFile = fs.createReadStream(nearFormLogoPath)
```
Comment (mcollina) [inline index.js:223]: remove

File: index.js [hunk 3]
```diff
@@ -268,7 +268,7 @@ class ClinicDoctor extends events.EventEmitter {
       headerLogo: logoFile,
       headerText: 'Doctor',
       toolVersion: doctorVersion,
-      nearFormLogo: nearFormLogoFile,
+      // nearFormLogo: nearFormLogoFile,
```
Comment (mcollina) [inline index.js:271]: remove

File: index.js (hunk 1 above)
Comment (jasnell) [inline index.js:151]:
```suggestion
```

File: index.js (hunk 2 above)
Comment (jasnell) [inline index.js:223]:
```suggestion
```

File: index.js (hunk 3 above)
Comment (jasnell) [inline index.js:271]:
```suggestion
```

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**